docs/articles/basics/first_bot.md

```diff
@@ -143,6 +143,24 @@ single log message from DSharpPlus. Woo hoo!
 ## Spicing Up Your Bot
 Right now our bot doesn't do a whole lot. Let's bring it to life by having it respond to a message!
 
+As of September 1st 2022, Discord started requiring message content intent for bots that want to read message content. This is a privileged intent!
+
+If your bot has under 100 guilds, all you have to do is flip the switch in the developer dashboard. (over at https://discord.com/developers/applications)
+If your bot has over 100 guilds, you'll need approval from Discord's end.
+
+After enabling the intent on Discords end, you have to specify your intents in you DiscordConfiguratioon:
```

Comment:
typo in DiscordConfiguration, also probably should use "in developer dashboard", instead of "on Discords end"